index.cjs

```diff
@@ -3185,11 +3189,7 @@ class BlockchainServiceBase {
                 gasPrice = await web3Instance.eth.getGasPrice();
             } else if (blockchain.name.startsWith('gnosis')) {
                 const response = await axios.get(blockchain.gasPriceOracleLink);
-                if (blockchain.name.split(':')[1] === '100') {
-                    gasPrice = Number(response.data.result, 10);
-                } else if (blockchain.name.split(':')[1] === '10200') {
-                    gasPrice = Math.round(response.data.average * 1e9);
-                }
+                gasPrice = Number(response.data.average) * 1e9;
```

Comment:
```suggestion
                gasPrice = Number(response?.data?.average) * 1e9;
```

Reply:
i disagree here actually, we should wrap it in try catch like u said and if response/data are undefined then we just go into catch which is good. otherwise who tf knows what js will do with Number(undefined), i dont want to think about that bs

Reply:
It might not trow error but response changes and error is thrown here.

On the other point `Number(undefined)` returns `NaN` so we know what will happen. And it will be hanled with my suggested code bellow